typescript/ws-protocol/src/types.ts

```diff
@@ -188,6 +200,19 @@ export type Time = {
 export type ServiceCallResponse = ServiceCallPayload & {
   op: BinaryOpcode.SERVICE_CALL_RESPONSE;
 };
+export type FetchAssetSuccessResponse = {
+  op: BinaryOpcode.FETCH_ASSET_RESPONSE;
+  requestId: number;
+  status: FetchAssetStatus.SUCCESS;
+  data: DataView;
+};
+export type FetchAssetErrorResponse = {
+  op: BinaryOpcode.FETCH_ASSET_RESPONSE;
+  requestId: number;
+  status: FetchAssetStatus.ERROR;
+  errorMsg: string;
```

Comment:
maybe name this `error: string`? I find the "msg" abbreviation a bit too informal for this API.